### ISSUE TITLE
Function 'get' bug?

### ISSUE THREAD

**Issue by BercziSandor** [pixelb/crudini#74]
# 1: input file
# 2: Problem: 'get' for all doesn't gives all values: as.alma & av.alma are missing.
# 3-5: get works for the exact value.
  
```bash
~$ cat a.ini  # 1
alma = 12

[as]
alma = 12
korte = 15

[av]
alma = 12
~$ crudini --get --format=lines a.ini # 2
[ DEFAULT ] alma = 12
[ as ] korte = 15
[ av ]

~$ crudini --get a.ini "as" alma # 3
12
~$ crudini --get a.ini "av" alma # 4
12
~$ crudini --get a.ini "" alma # 5
12
```

**Comment by pixelb** [comment]
Indeed. This is due to the matching name (alma) in the default/global section.
The code has this comment:
```
# XXX: Note this strips an item from section
# if matching value also in default (global) section
```
I'll look into fixing it, thanks

**Comment by pixelb** [comment]
The reason for this is that global/DEFAULT entries auto propagate to all other sections.
Therefore entries in other sections that have the same value are redundant, and so not listed.

More problematically if we did remove this logic there is no way to distinguish default from explicit entries in iniparse,
so default entries would be listed for every section.

To address this we'd need iniparse to distinguish default from explicit entries.
This is discussed at https://stackoverflow.com/questions/2189288/how-to-exclude-defaults-from-python-configparser-items
